src/Microsoft.DotNet.Web.ProjectTemplates.2.1/content/StarterWeb-CSharp/Startup.cs

```diff
@@ -53,6 +53,12 @@ public void ConfigureServices(IServiceCollection services)
                 .AddEntityFrameworkStores<ApplicationDbContext>()
                 .AddDefaultTokenProviders();
 
+            services.ConfigureApplicationCookie(options => {
+                options.LoginPath = "/tfp/Identity/Account/Login";
+                options.LogoutPath = "/tfp/Identity/Account/Logout";
+                options.AccessDeniedPath = "/tfp/Identity/Account/AccessDenied";
```

Comment:
Again, what's the `tfp`?

Reply:
Its just an acronym that we need for the MSAL libraries to work when we add Identity as a Service on top of Identity. It'll likely go away in the future. It stands for Trust framework policy.